test/sql-tap/pragma.test.lua

```diff
@@ -8,7 +8,7 @@ test:do_catchsql_test(
 	[[
 		PRAGMA kek = 'ON';
 	]], {
-		1, "Pragma 'KEK' does not exist"
+        1, "Pragma 'kek' does not exist"
```

Comment:
Typo: I suggest to leave the original indent level.

Reply:
See above.